cmd/query/app/http_handler_test.go

```diff
@@ -306,9 +306,6 @@ func TestGetTrace(t *testing.T) {
 			require.NoError(t, err)
 			assert.Empty(t, response.Errors)
 
-			assert.Len(t, exporter.GetSpans(), 1, "HTTP request was traced and span reported")
-			assert.Equal(t, "/api/traces/{traceID}", exporter.GetSpans()[0].Name)
```

Comment:
?

Reply:
@yurishkuro we're not tracing the request anymore after removing `otelhttp.NewHandler`

Reply:
but I thought we removed otelhttp.NewHandler because OTEL helper provided one. We should still be tracing requests.

Reply:
@yurishkuro OTEL does provide it but that gets provided in query server. This test is just for the HTTP handler. This is one of the reasons why I was asking if its okay to remove `otelhttp.NewHandler` because now the old flow won't get traced.

Reply:
👍  makes sense